kubernetes/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterScheduler.scala

```diff
@@ -91,10 +108,10 @@ private[spark] class KubernetesClusterScheduler(conf: SparkConf)
       s"--class=${args.userClass}",
       s"--master=$kubernetesHost",
       s"--executor-memory=${driverDescription.mem}",
-      s"--conf=spark.executor.jar=$clientJarUri",
-      s"--conf=spark.executor.instances=$instances",
-      s"--conf=spark.kubernetes.namespace=$nameSpace",
-      s"--conf=spark.kubernetes.driver.image=$sparkDriverImage")
+      s"--conf spark.executor.jar=$clientJarUri")
+
+    submitArgs ++= conf.getAll.filter { case (name, _) => !confBlackList.contains(name) }
+      .map { case (name, value) => s"--conf ${name}=${value}" }
```

Comment:
It must have been omitted in one of the commits. I will add the change in my next commit. 
Thanks for pointing that out.